src/components/EmbeddedContent/EmbeddedContent.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const EmbeddedContent = ({ style: propStyle, src, title }) => {
+  const style = {
+    border: 0,
+    width: '350px',
+    height: '470px',
+    ...propStyle,
+  }
+
+  return <iframe style={style} src={src} title={title} />
+}
+
+export default EmbeddedContent
```

Comment:
Super lightweight wrapper around an `<iframe>` so might not even be necessary...